src/test/java/it/gov/pagopa/payhub/activities/service/SendMailServiceTest.java

```diff
@@ -0,0 +1,359 @@
+package it.gov.pagopa.payhub.activities.service;
+
+import it.gov.pagopa.payhub.activities.activity.utility.SendEmailIngestionFlowActivityImpl;
+import it.gov.pagopa.payhub.activities.dto.MailTo;
+import jakarta.mail.BodyPart;
+import jakarta.mail.MessagingException;
+import jakarta.mail.internet.MimeMessage;
+import jakarta.mail.internet.MimeMultipart;
+import jakarta.mail.internet.PreencodedMimeBodyPart;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.mail.MailSendException;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import org.springframework.mail.javamail.MimeMessageHelper;
+
+import java.io.File;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@Slf4j
+@SpringBootTest(
```

Comment:
if possible, let's avoid @SpringBootTest test: to create a Spring context for each test could be heavy.
let's prefer to create real unit test, where you will create explicitly the object to be tested, injecting mocks through constructor (let's not change it this time, but remember it for next tests)